src/views/preview/comment/compose-comment.jsx

```diff
@@ -46,7 +46,8 @@ class ComposeComment extends React.Component {
             status: ComposeStatus.EDITING,
             error: null,
             appealId: null,
-            muteOpen: false
+            muteOpen: false,
+            muteExpiresAt: this.props.muteStatus.muteExpiresAt
```

Comment:
Thinking about dependencies on backend code a bit more, I think this depends on the session changes. For more resiliency, perhaps I should change this line to this: 
`this.props.muteStatus ? this.props.muteStatus.muteExpiresAt : 0`

or I could change the mapStateToProps to check whether muteStatus exists and then assign a default object. e.g. 
```
mute_status {
  muteExpiresAt: 0
  offenses: []
}
```

Thoughts?

Reply:
I think I like the second option better, but it would be fine to leave for a later PR if we're going to need updates based on upcoming backend changes.

Reply:
Done!